src/Playwright/Core/BrowserContext.cs

```diff
@@ -453,9 +453,9 @@ private void Channel_OnPage(object sender, BrowserContextPageEventArgs e)
 
         private void Channel_BindingCall(object sender, BindingCallEventArgs e)
         {
-            if (_bindings.TryGetValue(e.BidingCall.Name, out var binding))
+            if (_bindings.TryGetValue(e.BindingCall.Name, out var binding))
             {
-                _ = e.BidingCall.CallAsync(binding);
+                e.BindingCall.CallAsync(binding).ConfigureAwait(false);
```

Comment:
There I'd say we should intentionally not ignore the exception and it should lead to an unhandled exception.

Reply:
I don't think you need .ConfigureAwait(false) if you don't await.